### PR TITLE
feat(ui5-checkbox): adjusted to new Horizon theme specification

### PR DESCRIPTION
**Pull request by plamenivanov91**

### CODE REVIEW

File: packages/main/src/themes/CheckBox.css
Comment (kgogov) [inline packages/main/src/themes/CheckBox.css:1]: On line 38 the CSS declaration should be: `color: var(--sapField_TextColor)`

Comment (kgogov) [inline packages/main/src/themes/CheckBox.css:1]: In CheckBox.css on line 38 the declaration should be: `color: var(--sapField_TextColor)`

File: packages/main/src/themes/sap_horizon/CheckBox-parameters.css
Comment (kgogov) [inline packages/main/src/themes/sap_horizon/CheckBox-parameters.css:1]: In Horizon Light and Dark on lines 14 and 19 I think the declaration should be:
`--_ui5_checkbox_hover_background: var(--sapContent_Selected_Hover_Background);`

Comment (kgogov) [inline packages/main/src/themes/sap_horizon/CheckBox-parameters.css:1]: Also I saw duplicate styles for `--_ui5_checkbox_hover_background: var(--sapField_Hover_Background);`

**Comment by ilhan007** [comment]
Could you also add some before/after screenshots of the checkbox so one can grasp the Horizon update by looking at the PR description

**Comment by plamenivanov91** [comment]
> Could you also add some before/after screenshots of the checkbox so one can grasp the Horizon update by looking at the PR description

Here are the noticeable implemented differences:
[Check_Box_Visual Design.pdf](https://github.com/SAP/ui5-webcomponents/files/11333412/Check_Box_Visual.Design.pdf)
[Check_Box_Interaction Design.pdf](https://github.com/SAP/ui5-webcomponents/files/11333413/Check_Box_Interaction.Design.pdf)

**Review by kgogov — COMMENTED** [review]
All looks good to me.

**Review by kgogov — APPROVED** [review]

**Comment by petyabegovska** [comment]
Relates to #7000

**Pull request merged**